manifests/params.pp

```diff
@@ -14,7 +14,8 @@
 
   $context = 'i2b2',
   $local_base_url = 'http://localhost:8080',
-  $external_base_url = "http://${::fqdn}:8080",
+  $external_hostname = $::fqdn,
+  $external_base_url_tomcat = "http://${::fqdn}:8080",
```

Comment:
The project manager cell advertises the public URLs of the other cells. These urls are built from `$params::external_url`, which is in its turn built from this `$external_base_url_tomcat`.